.gitignore

```diff
@@ -0,0 +1,5 @@
+print/attendees.txt
+print/cookies.txt
+print/valid-attendees.txt
+.vagrant
```

Comment:
They are output that might contain attendee data and shouldn't be checked in.  Any reason not to have them ignored?